src/language/lang_de.php

```diff
@@ -0,0 +1,659 @@
+    'INAKTIV',
+  'PTS' =>
+    'PUNKTE',
+  'category' =>
+    'Kategorie',
+  'capture_' =>
+    'erobern_',
+  'Insert your answer' =>
+    'Gebe deine Antwort ein',
+  'Request Hint' =>
+    'Hinweis anfragen',
+  'Submit' =>
+    'Absenden',
+  'hint_' =>
+    'hinweis_',
+  'first_capture' =>
+    'erste_eroberung',
+  'completed_by' =>
+    'vervollständigt_von',
+  'scoreboard_' =>
+    'punktetafel_',
+  'filter_' =>
+    'filter_',
+  'rank_' =>
+    'rang_',
+  'team_name_' =>
+    'team_name_',
+  'quiz_pts_' =>
+    'quiz_punkte_',
+  'flag_pts_' =>
+    'flaggen_punkte_',
+  'base_pts_' =>
+    'basis_punkte_',
+  'total_pts_' =>
+    'gesamtpunkte_',
+  'team_' =>
+    'team_',
+  'team_members' =>
+    'team_mitglieder',
+  'base_pts' =>
+    'basis_punkte',
+  'quiz_pts' =>
+    'quiz_punkte',
+  'flag_pts' =>
+    'flaggen_punkte',
+  'total_pts' =>
+    'gesamtpunkte',
+  'Tool bars are located on all edges of the gameboard. Tap a category to expand and close each tool bar.' =>
+    'Symbolleisten befinden sich an allen Ecken des Spielbretts. Drücke auf auf eine Kategorie um diese zu öffnen und jede Symbolleiste zu schließen.',
+  'Tool_Bars' =>
+    'Tool_Bars',
+  'Tap the "Game Clock" to keep track of time during gameplay. Don’t let time get the best of you.' =>
+    'Drücke auf "Spielzeituhr" um die Zeit während des Spiels im Auge zu behalten.',
+  'Game_Clock' =>
+    'Spielzeituhr',
+  'Countries marked with an ' =>
+    'Länder markiert mit ',
+  'are captured by you.' =>
+    'sind von dir erobert.',
+  ' are owned by others.' =>
+    ' sind von anderen erobert.',
+  'Captures' =>
+    'Erobern',
+  'Tap Plus[+] to Zoom In. Tap Minus[-] to Zoom Out.' =>
+    'Drücke Plus[+] um hinein zu zoomen. Drücke Minus[-] heraus zu zoomen.',
+  'Click and Drag to move left, right, up and down.' =>
+    'Klicke und ziehe nach links, rechts, oben und unten um dich zu bewegen.',
+  'Zoom' =>
+    'Zoom',
+  'Tap Forward Slash [/] to activate computer commands. A list of commands can be found under "Rules".' =>
+    'Drücke [/] um Computerbefehle zu aktivieren. Eine Liste mit den Befehlen findest du unter "Regeln".',
+  'Command_Line' =>
+    'Kommandozeile',
+  'Click "Nav" to access main navigation links like Rules of Play, Registration, Blog, Jobs & more.' =>
+    'Drücke auf "Navigation" um auf die Hauptnavigationslinks wie z. B. Spielregeln, Registrierung, Blog, Jobs & mehr zu gelangen.',
+  'Track your competition by clicking "scorboard" to access real-time game statistics and graphs.' =>
+    'Verfolge deinen Wettkampf indem du auf "Punktetafel" klickst. Hier hast du Zugriff auf Spielstatistiken und -grafiken in Echtzeit.',
+  'Have fun, be the best and conquer the world.' =>
+    'Viel Spaß! Sei der Beste und erobere die Welt.',
+  'Game_On' =>
+    'Spiel starten',
+  'tutorial_' =>
+    'anleitung_',
+  'Next' =>
+    'Weiter',
+  'Skip to play' =>
+    'Überspringen',
+  'Powered By Facebook' =>
+    'Powered By Facebook',
```

Comment:
Doesn't seem like there was a translation added. I'd suggest "Präsentiert von Facebook" (Presented by Facebook) as that's generally the best way to phrase it in the language.